### PR TITLE
Add more tests to check for test coverage, current status 100%

### PR DESCRIPTION
**Pull request by manhlx3006**

### CODE REVIEW

File: test/v5/kyberDao.js [hunk 1]
```diff
@@ -1060,7 +1063,10 @@ contract('KyberDAO', function(accounts) {
                 );
                 assert(false, "throw was expected in line above.")
             } catch (e) {
-                assert(Helper.isRevertErrorMessage(e), "expected throw but got: " + e);
+                assert(
+                    Helper.isRevertErrorMessageContains(e, "validateParams: start in the past"),
```
Comment (ilanDoron) [inline test/v5/kyberDao.js]: please move to "exceptRevert" format with open zeppelin helper

File: test/v5/kyberDao.js [hunk 2]
```diff
@@ -5678,6 +5846,146 @@ contract('KyberDAO', function(accounts) {
...
             )
         });
+
+        it("Test constructor should revert invalid arguments", async function() {
+            let stakingContract = await StakingContract.new(kncToken.address, 10, currentBlock + 50, campCreator);
+
+            // epoch period is 0
+            try {
+                await DAOContract.new(
+                    0, currentBlock + 50,
```
Comment (ilanDoron) [inline test/v5/kyberDao.js]: please move to exceptRevert

**Comment by manhlx3006** [comment]
Added expectRevert and expectEvent for DAO tests will add for Staking later

File: test/v5/kyberDao.js [hunk 3]
```diff
@@ -1246,12 +1234,14 @@ contract('KyberDAO', function(accounts) {
 
         it("Test submit campaign should revert invalid campaign type", async function() {
             await deployContracts(30, currentBlock + 50, 10);
+            currentBlock = await Helper.getCurrentBlock();
+            // note: it is reverted as invalid opcode for campaign type, no message here
```
Comment (ilanDoron) [inline test/v5/kyberDao.js]: except revert doesn't work without a specific message?

**Pull request merged**